configs/base_config.py

```diff
@@ -15,20 +15,32 @@ def AWS_REGION(self):
         return self._get_config_value("AWS_REGION")
 
     @property
-    def AWS_OPEN_SEARCH_INDEX(self):
-        return self._get_config_value("AWS_OPEN_SEARCH_INDEX")
+    def AWS_RDS_DB_HOST(self):
+        return self._get_config_value("AWS_RDS_DB_HOST")
 
     @property
-    def AWS_OPEN_SEARCH_HOST(self):
-        return self._get_config_value("AWS_OPEN_SEARCH_HOST")
+    def AWS_RDS_DB_NAME(self):
+        return self._get_config_value("AWS_RDS_DB_NAME")
 
     @property
-    def AWS_OPEN_SEARCH_USERNAME(self):
-        return self._get_config_value("AWS_OPEN_SEARCH_USERNAME")
+    def AWS_RDS_DB_USERNAME(self):
+        return self._get_config_value("AWS_RDS_DB_USERNAME")
 
     @property
-    def AWS_OPEN_SEARCH_PASSWORD(self):
-        return self._get_config_value("AWS_OPEN_SEARCH_PASSWORD")
+    def AWS_RDS_DB_PASSWORD(self):
+        return self._get_config_value("AWS_RDS_DB_PASSWORD")
+
+    @property
+    def AWS_RDS_DB_PORT(self):
+        return self._get_config_value("AWS_RDS_DB_PORT")
+
+    @property
+    def SQLALCHEMY_DATABASE_URI(self):
+        return (
+            f'postgresql+psycopg2://{self._get_config_value("AWS_RDS_DB_USERNAME")}:'
```

Comment:
You dont need to get the value again, you can just access the properties:

```
f'postgresql+psycopg2://{self.AWS_RDS_DB_USERNAME}:'
f'{self:AWS_RDS_DB_PASSWORD}:{self.AWS_RDS_DB_HOST}:'
f'{self.AWS_RDS_DB_PORT}/{self.AWS_RDS_DB_NAME}`
```


Alternatively, if we don't use any of the individual values throughout the codebase and only ever access `SQLALCHEMY_DATABASE_URI`, we _could_ get rid of the properties such as `AWS_RDS_DB_HOST` etc and just keep `SQLALCHEMY_DATABASE_URI` making the explicit calls to `_get_config_value`.

Reply:
amended to one SQLALCHEMY_DATABASE_URI to retrieve from parameter store or environment variable